### ISSUE TITLE
user directory v1 feedback

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#7334]
post via comments below

**Comment by owocki** [comment]
* [ ] grants opened filter doesnt work https://bits.owocki.com/kpubegn9
* [ ] date filters dont work https://bits.owocki.com/P8ubwB1Q
* [ ] boolean filters dont seem to work https://bits.owocki.com/DOuxJpry

**Comment by frankchen07** [comment]
- [ ] csv export only exports the first page, not the entire search query
- [ ] labeling when you add a filter is a little weird (exact hacks came up as start, end date)
- [ ] ability to do multiple filters in one search isn't possible

**Comment by androolloyd** [comment]
> * [ ]  grants opened filter doesnt work https://bits.owocki.com/kpubegn9
- will debug alongside other currently invalid search strings.
> * [ ]  date filters dont work https://bits.owocki.com/P8ubwB1Q
- needs a date analyzer setup for elastic search as well as a front end component for picking date/time
> * [ ]  boolean filters dont seem to work https://bits.owocki.com/DOuxJpry
- like the above the analyzer doesn't seem to detect these correctly, more debugging required, 
> * [ ]  can we add copy/paste-able links to the directory?  for example, i serach for foo=bar and foo1=bar1 and i get gitcoin.co/user_directory?foo=bar&foo1=bar1 in the URL bar. when someone loads that search they'll be taken to that page
- its doable, but requires hooking into the internal store of the innerSearch.js library, with the param values, can be done, just a matter of working through whats wrong.
> * [ ]  persona filter doesnt work https://bits.owocki.com/P8ubOq12
- there are a few strings that are not being picked up, why this is an unknown, "solidity" as a keyword is another example, perhaps the analyzers aren't indexing the entirety of the arrays,

**Comment by PixelantDesign** [comment]
Loads very fast!

- [ ] Could we import the current  user directory's existing css?
- [ ] Use no data styling from existing user directory
- [ ] Getting a  404 when I click to see the profile details
- [ ] It was unexpected that I clicked **export button** and the csv file downloaded but didn't tell me where the file was downloaded to. Maybe ok  since it's an internal tool for right now.  
- [ ] Search terms aren't displaying any results ( I searched Arweave, near, gitcoin)
<img width="1579" alt="Screen Shot 2020-09-24 at 12 59 29 AM" src="https://user-images.githubusercontent.com/238235/94103052-96675c00-fe01-11ea-8fb5-113148579f52.png">
<img width="1568" alt="Screen Shot 2020-09-24 at 12 59 12 AM" src="https://user-images.githubusercontent.com/238235/94103063-9a937980-fe01-11ea-8c89-7dffad348dcc.png">
